vomsimporter.py

```diff
@@ -182,7 +186,9 @@ def find_group_by_name(self, group_name):
         if total_results == 0:
             return None
         if total_results == 1:
-            return data['Resources'][0]
+            group = data['Resources'][0]
+            self._iam_groups[group['displayName']] = group
```

Comment:
For me it would be more clear to use `self._iam_groups[group_name] = group`, because with returned record `group['displayName']` it is necessary to know how IAM internally parameter passed to `/iam/group/find/byname` endpoint.

Reply:
Ok.